lib/options-parser.js

```diff
@@ -44,6 +46,7 @@ function parseOptions(argsOriginal) {
     parsed.url = args[0];
     parsed.css = args[1];
 
+    // this is problematic...
```

Comment:
You are right - we do need to remove the check ... 

```
$ phantomjs penthouse.js ~/dev_priv/penthouse/test/static-server/page1.html  ~/dev_priv/penthouse/test/static-server/page1.css 
#box1 {
    width : 800px;
    height: 400px;
    background : red;
}
#box2 {
    width : 800px;
    height: 400px;
    background : blue;
}

#box21 {
    width : 100%;
    height: 100px;
    background: yellow;
}

#box22 {
    width : 100%;
    height: 50px;
    padding-top: 200px;
    background: green;
}
```
There is no need to fire up a server, local file paths work just fine.